### DISCUSSION TITLE
VS Extension's Context menu not showing existing steps with provided parameter values

### DISCUSSION THREAD

**Discussion by ddizmang** [reqnroll/Reqnroll#216] [Ideas]
I am returning to C#/ReqnRoll after a couple of years, I thought that SpecFlow showed existing steps with parameters from other feature files in the extension's context menu (Intellisense).

example, if I had a step defined as:
Given the API URL is (.*)

With an existing feature scenario with the following:
Given the API URL is http://test.com

Then in the VS extension's context menu, I would see both steps as options like:
Given the API URL is (.*)
Given the API URL is http://test.com

Was the functionality removed or is it allowed with a setting?

**Comment by gasparnagy** [comment]
This feature has been "removed" from SpecFlow with VS2022 (to be precise was not re-implemented when we re-implemented the entire VS extension), so Reqnroll does not have it either.

The way it was originally implemented was too resource-heavy as it kept all feature files in memory. Re-implementing that in a better way is not a trivial task, although not impossible.

Would be useful though, so I re-classify this topic as an "idea".
